lib/auth.ts

```diff
@@ -210,3 +210,5 @@ export class ResinAuth {
 		});
 	};
 }
+
+module.exports = ResinAuth;
```

Comment:
I think the way to do it in TypeScript is `export = ResinAuth`.

You can do it above as well as `export = class ResinAuth { ... }` but doing it in 2 steps is fine as well.

Reply:
That's to maintain the same import approach. I read in FD that we might go the `export default` way, then this ^ doesn't apply